.github/workflows/tests.yml

```diff
@@ -223,3 +223,35 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: gradle/wrapper-validation-action@v1
+  checkauthors:
+    name: "Validate AUTHORS"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # full checkout needed for authors generation
+          fetch-depth: 0
+      - name: "No new AUTHORS"
+        id: authors_check
+        shell: bash
+        run: |
+          ./scripts/generate-authors.sh
+          set +o pipefail
+          added=$(git diff HEAD --no-ext-diff --unified=0 -a --no-prefix | egrep "^\+[^+]" | sed "s/^\+//")
+          if [ -z "$added" ]; then
+            echo "::set-output name=newauthor::false"
+            echo "No authors added"
+            exit 0
+          fi
+          message="When merged, this pull request will introduce additions in the [AUTHORS file](https://github.com/JabRef/jabref/blob/master/CONTRIBUTING.md#author-credits). In case you want to appear differently, please [adjust your name in your git configuration](https://docs.github.com/en/github/using-git/setting-your-username-in-git).%0A%0A\`\`\`%0A${added//$'\n'/'%0A'}%0A\`\`\`%0A"
```

Comment:
And then force push? or how is the change to the git config reflected in the PR?

Reply:
Yes. Could be difficult for the newcomers. Nevertheless, I would urge them to get their git configured properly.

I added some text with links here.